src/main/java/org/candlepin/subscriptions/capacity/CapacityProductExtractor.java

```diff
@@ -48,9 +50,32 @@ public CapacityProductExtractor(ProductIdToProductsMapSource productIdToProducts
     }
 
     public Set<String> getProducts(Collection<Integer> productIds) {
-        // NOTE: this logic can grow over time (e.g. to blacklist certain combinations, etc).
-        return productIds.stream()
+        Set<String> products = productIds.stream()
             .map(productIdToProductsMap::get).filter(Objects::nonNull).flatMap(List::stream)
             .collect(Collectors.toSet());
+
+        if (setIsInvalid(products)) {
+            return Collections.emptySet();
+        }
+
+        return products;
     }
+
+    /**
+     * Return whether this set of products should be considered for capacity calculations.
+     * @param products a set of product names
+     * @return true if the set is invalid for capacity calculations
+     */
+    public boolean setIsInvalid(Set<String> products) {
+        return products.stream().anyMatch(matchesRhel()) && products.stream().anyMatch((matchesSatellite()));
+    }
+
+    private Predicate<String> matchesRhel() {
+        return x -> x.matches("^RHEL.*");
+    }
+
+    private Predicate<String> matchesSatellite() {
+        return x -> x.matches("^Satellite.*");
```

Comment:
For this to work, we need to add `Satellite 6` and `Satellite 6 Capsule` to the product ID mapping. The Eng IDs we should use for this are 250 (Satellite) and 269 (Capsule).

Reply:
I also just realized we can avoid regex here; this can just be `x.startsWith('Satellite')`